From 17ccc945fbdff51a7e9415fef0c1a2bf509319e1 Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Wed, 15 Jul 2020 10:09:31 +0200
Subject: [PATCH] Disable some plugins

- The updateinfo plugin only works when
using the installer

- The clangformat plugin requires a clang patch
which was never accepted upstream [1]

- The qlitehtml plugin causes installation issues.
The documentation can already be browsed using
either QtWebEngine or QTextBrowser.

[1] https://reviews.llvm.org/D53072
---
 src/plugins/CMakeLists.txt      | 2 --
 src/plugins/help/CMakeLists.txt | 9 ---------
 2 files changed, 11 deletions(-)

diff --git a/src/plugins/CMakeLists.txt b/src/plugins/CMakeLists.txt
index 871f999b..e6d82385 100644
--- a/src/plugins/CMakeLists.txt
+++ b/src/plugins/CMakeLists.txt
@@ -7,7 +7,6 @@ add_subdirectory(serialterminal)
 add_subdirectory(helloworld)
 add_subdirectory(imageviewer)
 add_subdirectory(marketplace)
-add_subdirectory(updateinfo)
 add_subdirectory(welcome)
 
 # Level 2: (only depends on Level 1 and below)
@@ -41,7 +40,6 @@ add_subdirectory(vcsbase)
 # Level 5:
 add_subdirectory(bazaar)
 add_subdirectory(beautifier)
-add_subdirectory(clangformat)
 add_subdirectory(clangrefactoring)
 add_subdirectory(clearcase)
 add_subdirectory(cvs)
diff --git a/src/plugins/help/CMakeLists.txt b/src/plugins/help/CMakeLists.txt
index 2063293d..4f836c93 100644
--- a/src/plugins/help/CMakeLists.txt
+++ b/src/plugins/help/CMakeLists.txt
@@ -60,15 +60,6 @@ extend_qtc_plugin(Help
     webenginehelpviewer.h
 )
 
-if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/qlitehtml/litehtml/CMakeLists.txt)
-  add_subdirectory(qlitehtml)
-else()
-  find_package(litehtml QUIET)
-  if (TARGET litehtml)
-    add_subdirectory(qlitehtml)
-  endif()
-endif()
-
 extend_qtc_plugin(Help
   CONDITION TARGET litehtml AND TARGET qlitehtml
   FEATURE_INFO "litehtml help viewer"
-- 
2.29.2

